packages/constants-internal/src/constants.ts

```diff
@@ -283,3 +283,28 @@ export const STATUS = {
   SUCCESS: 'success',
   ERROR: 'error',
 };
+
+/**
+ * Constants representing the help text for
+ * each view.
+ */
+export const ViewHelpMapping = {
+  SCATTERPLOT: {
+    PCA: 'A 2D Scatterplot of the cell embeddings extracted using the Principal Component Analysis (PCA) technique',
+    TSNE: 'A 2D Scatterplot of the cell embeddings extracted using the t-distributed Stochastic Neighbor Embedding (t-SNE) technique',
+  },
+  HEATMAP: 'This Heatmap View provides an overview of the gene expression (represented by color) of the genes (i.e., rows) in each cell (i.e., column). The cellSet row on top shows the cells in each cluster (color mapped to clusters in scatterplot)',
```

Comment:
> gene expression ... genes ... cell ... cellSet ...cells

Since Vitessce can handle features and observations very generally (not necessarily always genes and cells), we might want to consider obtaining the default help text via a function like

`generateHelpText(coordinationValues: { obsType, featureType, etc. }) -> string` (pseudocode) (while also keeping it possible for users to potentially provide a plain string `helpText` prop to each `_Subscriber` component that would override the more dynamic function result).

Reply:
This does not necessarily need to be done in this PR - we could create a new issue so that it could be done in a follow-up pull request. Happy to discuss in a call as well.